docs/DeveloperGuide.md

```diff
@@ -158,6 +158,7 @@ This section describes some noteworthy details on how certain features are imple
 
 #### About this feature
 
+The add internship application feature allows users to quickly add an internship application  in the tracker via the
```

Comment:
```suggestion
The add internship application feature allows users to quickly add an internship application in the tracker via the
```
Changing this back

Reply:
Thanks for pointing that out, will be fixed.